NewForce-Cohort-9/tabloidmvc-dalvin-and-the-chipmunks
# #11 Delete a Category

# Description

When an admin is on the Tag list page, they can select the Create Category button to be directed to a form in which they can delete a Category name. Saving the Category will redirect the admin to the updated list of Categories. [Trello Ticket #11](https://trello.com/c/mD2OuQni/11-11-delete-a-category)

## Type of change



# Testing Instructions

From the command line:
```git add``` and ```git commit``` whatever you're currently working on on your branch
```git checkout main```
```git fetch --all```
```git checkout bp3-delete```
If you don't see the most recent version of my files, ```git pull origin bp3-delete```
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloidmvc-dalvin-and-the-chipmunks/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloidmvc-dalvin-and-the-chipmunks/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
Click the execute with debugger solid green button to launch the app
In the browser, please go to Login on the nav bar and log in with this email: [admin@example.com](mailto:admin@example.com)
(This should redirect you back to the home page.)
Then:

* Click Category Management in the navbar 
* On the Category Management page, click New Category 
* After inputting a name like "Testing", click Save
* You should be redirected back to the Categories list, where the new Category, named "Testing", should be displayed in the Category List under Technology
* Once you have confirmed that it is displaying properly, click on Delete to the far-right of your created category
* You will be redirected to form a form that asks "Are you sure you want to delete this Category?"
* After clicking Delete you will be taken back to the Category Management page



# Checklist:
